src/components/LiquidityChartRangeInput/index.tsx

```diff
@@ -157,7 +157,7 @@ export default function LiquidityChartRangeInput({
   )
 
   if (error) {
-    sendEvent('exception', { description: error.toString(), fatal: false })
+    Sentry.captureMessage(error.toString(), 'log')
```

Comment:
Don't we also want to capture any stack in the `error`?

Reply:
I am actually not sure whether we want to track this and the other error outside of the ErrorBoundary. This is inconsistent in a sense that we're not tracking other errors throughout the app in a similar way. 

Answering your question, I take this error as a backend error and that's probably why we never tracked its stack trace.

Unless we find that information necessary, I would remove it.